hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileAppendTest.java

```diff
@@ -198,6 +212,40 @@ void appliesUpgradeFileNewContent() {
         assertEquals(bytesNewContentExpected, file);
     }
 
+    @Test
+    @DisplayName("Fails handle if keys doesn't exist on file to be appended")
+    void keysDoesntExist() {
```

Comment:
Might be good to rename this to "failsForImmutableFile" to make clearer that we're testing how the code handles immutable files, not verifying that keys do not exist.

Reply:
done